invokeai/frontend/web/src/features/ui/hooks/usePanel.ts

```diff
@@ -125,15 +125,26 @@ export const usePanel = (arg: UsePanelOptions): UsePanelReturn => {
 
       _setMinSize(minSizePct);
 
-      // Resize if the current size is smaller than the new min size - happens when the window is resized smaller
-      if (panelHandleRef.current.getSize() < minSizePct) {
+      const currentSize = panelHandleRef.current.getSize();
+
+      // If currentSize is 0, the panel is collapsed, so we don't want to resize it
+      // If it's not 0, but less than the minSize, resize it
+      if (currentSize && currentSize < minSizePct) {
```

Comment:
Will it ever be undefined?

Reply:
No, it will always be a number.

Reply:
Updated the comparison here to be clearer:
```ts
if (currentSize > 0 && currentSize < minSizePct) {
```